java/java-impl/src/META-INF/JavaPlugin.xml

```diff
@@ -237,6 +237,9 @@
     <extensionPoint qualifiedName="com.intellij.debugger.asyncStackTraceProvider"
                     interface="com.intellij.debugger.engine.AsyncStackTraceProvider" dynamic="true"/>
 
+    <extensionPoint qualifiedName="com.intellij.debugger.kotlinStackFrameValueContributor"
+                    interface="org.jetbrains.kotlin.idea.debugger.core.stackFrame.KotlinStackFrameValueContributor" dynamic="true"/>
```

Comment:
By the way, there is `com.intellij.debugger.engine.FrameExtraVariablesProvider`. Is it not enough for your use-cases?

Reply:
I wasn't aware of this EP but unfortunately, it's not enough for my use case.

I think it should be possible to extend FrameExtraVariablesProvider in a way that will make it work.

I'm sending you an image of my use case in a private channel.

Reply:
My use requires adding a custom XValue to children. The existing EP doesn't allow that. It just seems allow adding a code fragment dressed as a variable.

As I said, I think I can extend it by adding another method to to that returns a list or XValue that can be added to children.

Do you want me to make this change or stick with my new EP?